pkg/debuggers/local/java_port.go

```diff
@@ -0,0 +1,27 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+type JavaPortInterface struct{}
+
+func (g *JavaPortInterface) GetRemoteConnectionCmd(plankName, plankNamespace, podName, podNamespace string, localPort, remotePort int) *exec.Cmd {
+	// connect directly to the container we are debugging
+	return GetPortForwardCmd(podName, podNamespace, localPort, remotePort)
+}
+
+func (d *JavaPortInterface) GetDebugCmd(localPort int) *exec.Cmd {
+	fmt.Printf("Java debug port available on local port %v.\n", localPort)
+	cmd := exec.Command("sleep", "200000")
```

Comment:
maybe add a TODO to unhack this

Reply:
https://github.com/solo-io/squash/issues/86